src/cartesi-machine.lua

```diff
@@ -437,9 +437,9 @@ local memory_range_replace = {}
 local ram_image_filename = images_path .. "linux.bin"
 local ram_length = 64 << 20
 local dtb_image_filename = nil
-local dtb_bootargs = "console=hvc0 rootfstype=ext2 root=/dev/pmem0 rw quiet \z
-                      swiotlb=noforce init=/opt/cartesi/bin/init"
```

Comment:
I checked that `swiotlb=noforce` is actually not needed anymore to workaround the memory usage by kernel itself when RAM >= 128MB, so I decided to remove it to simplify bootargs.

Reply:
So, this was only an issue when memory was small, right? We didn't want to waste memory reserving it for DMA stuff that we never use. When memory is greater than 128 they do not reserve? Is that the change? Or when memory is less than 128 they don't reserve?

Reply:
Issue appeared when ram was 128MB or larger.
@edubart what kernel did you try removing it from?

commit: https://github.com/cartesi/machine-emulator/commit/525e152b32f954f9a679b272e51012e860e5e00b
ref: http://xillybus.com/tutorials/iommu-swiotlb-linux

Reply:
@mpolitzer I tried kernel 6.1.x and 5.15.63, with different ram sizes, and I could not trigger the issue reported in https://github.com/cartesi-corp/machine-emulator/issues/439 . The kernel config today has many differences than the one at the time, I'm not sure why is not happening anymore, but since it is not happening, I decided to simplify.